packages/flutter_tools/lib/src/intellij/intellij_validator.dart

```diff
@@ -482,6 +483,16 @@ class IntelliJValidatorOnMac extends IntelliJValidator {
           ]),
       ));
     }
+
+    // Remove JetBrains Toolbox results
```

Comment:
Consider explaining _why_ we are removing these results. This is more important than explaining _what_ we are removing.

```suggestion
    // Remove JetBrains Toolbox link apps. These tiny apps just
    // link to the full app, will get detected elsewhere in our search.
```